Dynavity/Dynavity/view-model/CanvasSelectionViewModel.swift

```diff
@@ -57,6 +57,10 @@ class CanvasSelectionViewModel: ObservableObject {
         self.objectWillChange.send()
     }
 
+    func importCanvas(name: String, owner: String) -> Bool {
+        canvasRepo.importCanvas(ownerId: owner, canvasName: name)
```

Comment:
Otherwise, the UI will not update after importing
```suggestion
        canvasRepo.importCanvas(ownerId: owner, canvasName: name)
        self.objectWillChange.send()
```

Can help to add the above change to `CanvasViewModel`'s `saveCanvas` as well?